src/blockchain/pop/vbk_block_tree.cpp

```diff
@@ -86,15 +86,8 @@ bool VbkBlockTree::acceptBlock(const VbkBlock& block,
 
   assert(index != nullptr);
 
-  // second, do stateless validation of payloads
-  for (const auto& p : payloads) {
-    if (!checkPayloads(p, state, getParams(), btc().getParams())) {
-      return state.Invalid("vbk-check-payloads");
-    }
-  }
-
-  if (!cmp_.proceedAllPayloads(*index, payloads, state)) {
-    return state.Invalid("VbkTree::acceptBlock");
+  if (!cmp_.proceedAllEndorsements(*index, context, state)) {
+    return state.Invalid("vbk-procedd-all-endorsements-block");
```

Comment:
proceed

Reply:
@blackyblack it will be fixed in next PR